docs/faq/help.txt

```diff
@@ -26,7 +26,7 @@ Then, please post it in one of the following channels:
   chat-based discussions.
 
 .. _`"Using Django"`: https://forum.djangoproject.com/c/users
-.. _`#django IRC channel`: irc://irc.freenode.net/django
+.. _`#django IRC channel`: https://freenode.net/kb/answer/chat
```

Comment:
I wonder if we can use `https://kiwiirc.com/nextclient/irc.freenode.net/django` instead. It redirects to the specific channel.

Reply:
I've changed it to your suggested URL.

Reply:
We don't need to point to kiwiirc.com for this. We can use freenode's hosted version: https://webchat.freenode.net/#django 

I liked the idea of the freenode page since that specifies all the different ways of joining, and informs users a lot. Perhaps we should really have a page in the docs saying what IRC is, linking to the freenode "how to connect page", *and* the individual channels' webchat.freenode.net links. Could be in FAQ?

Reply:
> https://webchat.freenode.net/#django

True, let's use it.

> I liked the idea of the freenode page since that specifies all the different ways of joining, and informs users a lot. Perhaps we should really have a page in the docs saying what IRC is, linking to the freenode "how to connect page", and the individual channels' webchat.freenode.net links. Could be in FAQ?

IMO folks who use IRC will know what to do, and we'll redirect others directly to the channel. I think we can add a short sentence to the `docs/faq/help`, e.g.
```
See the `Freenode documentation <https://freenode.net/kb/answer/chat>`_ if you're new to IRC.
```

Reply:
Yeah I think that’s fair. I suppose if one uses IRC webchat enough, one finds out about what IRC really is under the hood.

Reply:
Thanks @adamchainz for your suggestion.
@felixxm I've changed the link and added your suggested sentence to `docs/faq/help` in a separate commit.